src/FantomasOptionsPage2.xaml

```diff
@@ -0,0 +1,40 @@
+﻿<UserControl
+    x:Class="FantomasVs.FantomasOptionsPage2"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    d:DesignHeight="450"
+    d:DesignWidth="800"
+    mc:Ignorable="d">
+    <Grid>
+
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="2*" />
+            <RowDefinition Height="*" />
+        </Grid.RowDefinitions>
+
+        <TextBlock
+            Grid.Row="0"
+            Text="Displays settings as defined in the .editorconfig or if no settings can be found the user profile's formatting options."
+            TextWrapping="Wrap" />
+
+        <Button
+            Grid.Row="1"
+            Margin="0,5"
+            Padding="5"
+            HorizontalAlignment="Left"
+            Content="Generate .editorconfig file from settings" />
+
+        <DataGrid Grid.Row="2">
+            <!--  Here goes options  -->
+        </DataGrid>
+
+        <Grid Grid.Row="3" Margin="0,5,0,0">
+            <!--  Here goes code example  -->
```

Comment:
Shall we add a new submodule to integrate your code visualizer control or are you going to publish a nuget?

Reply:
Did you just read my mind?